### PR TITLE
best branch name ever

### PR DESCRIPTION
**Pull request by antonbobkov**

### CODE REVIEW

File: src/pop_protocol.erl [hunk 1]
```diff
@@ -237,3 +248,28 @@ resolve_fork_same_parent(B1, B2) ->
...
+    #protocol_data{
+	    verifiers_arr = VerifierArr,
+	    time_between_blocks = TimeBetweenBlocks,
+	    last_block = LastBlock
+      } = PD,
+
+    VerNum = array:size(VerifierArr),
+    
+    LastTime = get_block_cd(timestamp, LastBlock),
+    LastZeroTime = LastTime - (LastTime rem (VerNum * TimeBetweenBlocks)),
```
Comment (pdlla) [inline src/pop_protocol.erl]: such math 😱

File: src/pop_protocol.erl [hunk 2]
```diff
@@ -222,3 +248,28 @@ resolve_fork_same_parent(B1, B2) ->
...
+    NextTime = LastZeroTime + VerifierIndex * TimeBetweenBlocks,
+
+    if 
+	NextTime =< LastTime ->
```
Comment (antonbobkov) [inline src/pop_protocol.erl]: =<

**Comment by pdlla** [comment]
merge merge!

**Comment by pdlla** [comment]
I just figured out what "Close" means in a pull request D:

**Pull request merged**